src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -382,10 +387,24 @@ def _cc_receive(
             log.exception(f"encountered error while receiving message via {self}")
             return {"msg": None}
 
-    # TODO: refactor to use trc reader when new version of python can is release
-    def _merge_trc(self) -> None:
-        """Merge multiple trc files in one."""
+    @staticmethod
+    def _extract_header(trace: Path):
+        """Extract data from trc file header
+        
+        :param trace: trc file from which to extract the data
+        
+        :return: header data"""
+        header_data = []
+
+        with trace.open("r") as trc:
+            data = trc.read().splitlines(True)
+            for line in data:
+                if line.startswith(";"):
```

Comment:
Same here, if the header data stops as soon as there is no ; start character anymore, why continue through the rest of the file?